src/apps/monitoring/latencetech/restapi/mode/connectivity.pm

```diff
@@ -0,0 +1,420 @@
+                key_values => [ { name => 'expectedStabilityPercent' }, { name => 'display' } ],
+                output_template => 'Expected stability: %.2f%%',
+                perfdatas => [
+                    { value => 'expectedStabilityPercent', template => '%.2f',
+                      min => 0, max => 100, unit => '%', label_extra_instance => 1, instance_use => 'display' },
+                ],
+            }
+        },
+        { label => 'volatility-prct', nlabel => 'volatility.percentage', set => {
+                key_values => [ { name => 'volatilityPercent' }, { name => 'display' } ],
+                output_template => 'Volatility: %.2f%%',
+                perfdatas => [
+                    { value => 'volatilityPercent', template => '%.2f',
+                      min => 0, max => 100, unit => '%', label_extra_instance => 1, instance_use => 'display' },
+                ],
+            }
+        },
+        { label => 'qoe-rate', nlabel => 'qoe.rate.value', set => {
+                key_values => [ { name => 'qualityOfExperience' }, { name => 'display' } ],
+                output_template => 'Quality of Experience: %.3f',
+                perfdatas => [
+                    { value => 'qualityOfExperience', template => '%.3f',
+                      min => 0, max => 5, label_extra_instance => 1, instance_use => 'display' },
+                ],
+            }
+        },
+        { label => 'packet-loss-prct', nlabel => 'packetloss.rate.percentage', set => {
+                key_values => [ { name => 'packetLossRatePercent' }, { name => 'display' } ],
+                output_template => 'Packet loss rate: %.2f%%',
+                perfdatas => [
+                    { value => 'packetLossRatePercent', template => '%.2f',
+                      min => 0, max => 100, unit => '%', label_extra_instance => 1, instance_use => 'display' },
+                ],
+            }
+        },
+        { label => 'expected-packet-loss-prct', nlabel => 'expected.packetloss.rate.percentage', set => {
+                key_values => [ { name => 'expectedPacketLossPercent' }, { name => 'display' } ],
+                output_template => 'Expected packet loss rate: %.2f%%',
+                perfdatas => [
+                    { value => 'expectedPacketLossPercent', template => '%.2f',
+                      min => 0, max => 100, unit => '%', label_extra_instance => 1, instance_use => 'display' },
+                ],
+            }
+        },
+        { label => 'connectivity-health',
+            type => 2,
+            warning_default => '%{connectivityHealth} =~ "Warning"',
+            critical_default => '%{connectivityHealth} =~ "Need Attention"',
+            set => {
+                key_values => [ { name => 'connectivityHealth' }, { name => 'display' } ],
+                closure_custom_output => $self->can('custom_status_output'),
+                closure_custom_perfdata => sub { return 0; },
```

Comment:
```suggestion
```

Reply:
This should not be necessary when the type of the counter is 2. Your tests should all run OK without this line.